tests/lib/devicetree/devices/src/main.c

```diff
@@ -320,20 +320,13 @@ static void test_supports(void)
 	zassert_true(check_handle(DEV_HDL(TEST_PARTITION), hdls, nhdls), NULL);
 }
 
-void test_main(void)
+void *devicetree_api_setup(void)
```

Comment:
Suggested "devicetree_devices_setup" to match the testsuite.

Reply:
> Suggested "devicetree_devices_setup" to match the testsuite.

ok,I will do it as soon